app/controllers/posts_controller.rb

```diff
@@ -36,6 +36,17 @@ def create
     end
   end
 
```

Comment:
- [ ] Please make sure to use `CanCanCan` for Post authorization in `Post Controller`.
  - You can consider using the below code.
 
```ruby
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```